galactic/distribution.yaml

```diff
@@ -1342,6 +1342,17 @@ repositories:
       url: https://github.com/fzi-forschungszentrum-informatik/lanelet2.git
       version: master
     status: maintained
+  laser_filters:
+    release:
```

Comment:
you can add a `doc` entry here as well

Reply:
IIUC, for cpp projects it automatically generates doxygen documentation.
You might need a doxyfile for it to work correctly, e.g. https://github.com/ros2/rmw_fastrtps/blob/master/rmw_fastrtps_cpp/Doxyfile.

For python projects it does nothing at the moment.